main/ZgatewayBT.ino

```diff
@@ -611,6 +754,8 @@ bool BTtoMQTT() {
     if (returnedString.equals(F(BLEEndOfDiscovery))) //OK+DISCE
     {
       returnedString.remove(0); //clear data string
+      scanCount++;
+      Log.notice(F("Scan number %d end " CR), scanCount);
```

Comment:
why do we increase scanCount here?

Reply:
It is hidden in the commit, this is for the future publication of the scan count to SYStoMQTT